src/main/java/piano/SaveAudio/FileLocationGUI.java

```diff
@@ -0,0 +1,33 @@
+package piano.SaveAudio;
+
+import piano.main.PianoGUI;
+import piano.recorder.KeyPressedInfo;
+
+import javax.sound.midi.MidiChannel;
+import javax.swing.*;
+import java.io.File;
+import java.util.ArrayList;
+
+public class FileLocationGUI extends JFrame {
+    JFileChooser fileChooser;
+    PianoGUI pianoGUI;
+    File fileLocation;
+    AudioToMidi audioToMidi;
+
+    public FileLocationGUI(ArrayList<KeyPressedInfo> recordedKeys) {
+        setTitle("Save Audio");
+        setDefaultCloseOperation(EXIT_ON_CLOSE);
```

Comment:
Most likely you don't want exit on close

Reply:
I'm not sure which other close operation I should use. DoNothing and Hide seem to not be good solutions since the program calls a new FileLocationGui for each time you choose to save
Would Dispose be better?

Reply:
Dispose would be better.